Sources/_MatchingEngine/Regex/AST/CustomCharClass.swift

```diff
@@ -81,11 +84,25 @@ extension CustomCC.Member {
     case .range(let r): return r
     case .atom(let a): return a
     case .quote(let q): return q
+    case .trivia(let t): return t
     case .setOperation(let lhs, let op, let rhs): return (lhs, op, rhs)
     }
   }
 
   func `as`<T>(_ t: T.Type = T.self) -> T? {
     _associatedValue as? T
   }
+
+  public var isTrivia: Bool {
+    if case .trivia = self { return true }
+    return false
+  }
+}
+
+extension AST.CustomCharacterClass {
+  public var strippingTrivia: Self {
+    var copy = self
+    copy.members = copy.members.filter { !$0.isTrivia }
+    return copy
```

Comment:
Wouldn't this need to recursively strip?

Reply:
Not for the places where it's currently used, I've renamed it to `strippingTriviaShallow` for now to at least make it clear, and will look at adding a recursive version when looking at adding more general AST trivia stripping utilities